src/main/configurations/Translate/Configuration_GetRolTypenByUUID.xml

```diff
@@ -0,0 +1,47 @@
+<Module xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="../FrankConfig.xsd">
+    <Adapter name="GetRolTypenByUUID">
+        <Receiver>
+            <JavaListener name="GetRolTypenByUUID" />
+        </Receiver>
+        <Pipeline>
+            <Exits>
+                <Exit name="EXIT" state="SUCCESS" />
+                <Exit name="EXCEPTION" state="ERROR" />
+            </Exits>
+
+            <SenderPipe
+                name="GetRolTypenByUUIDSender"
+                getInputFromSessionKey="originalMessage"
+            >
+                <HttpSender
+                    name="GetRolTypenByUUIDHttpSender"
+                    methodType="GET"
+                    headersParams="Authorization,Accept-Crs"
+                    timeout="${zaakbrug.zgw.catalogi-api.timeout}"
+                    maxExecuteRetries="5"
+                />
+                <Param name="url" xpathExpression="/roltypen" />
+                <Param name="Accept-Crs" value="EPSG:4326" />
+                <Param name="Authorization" value="Bearer @@catalogi-api.jwt@@" />
+                <Forward name="success" path="JsonToXml" />
+                <Forward name="exception" path="ErrorJsonToXml" />
+            </SenderPipe>
+
+            <JsonPipe name="ErrorJsonToXml">
+                <Forward name="success" path="buildErrorMsg" />
+            </JsonPipe>
+            <XsltPipe name="buildErrorMsg"
+                styleSheetName="Common/xsl/ParseNegativeHttpResult.xsl">
+                <Param name="senderPipeName" value="GetZgwRolTypeByZaakTypeSender" />
+                <Forward name="success" path="EXCEPTION" />
+            </XsltPipe>
+
+            <JsonPipe
+                name="JsonToXml">
+                <Forward name="success" path="EXIT" />
+            </JsonPipe>
+
```

Comment:
In all adapters that do api calls, we end with a pipe that unwraps the response envelope and then wraps it in a root element with a typename we assign to it. In this case it would be "ZgwRolType". You can use the already existing XSLT UnwrapOpenZaakApiEnvelopeToSingle.xslt for this or just copy/paste it from any other ByUrl adapter.